Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/drinks_soda.yml

```diff
@@ -41,6 +41,7 @@
       - DrinkWaterCup
       - DrinkBeerCan
       - DrinkWineCan
+      - DrinkHotMoffee
```

Comment:
The soda machine is for cans, the hot drinks machine is moreso for cupped drinks. There are usually plenty of those types of vending machines around the station.

Reply:
This file is for the randomly-spawned vending machine drinks, which do include HotCoffee etc. Not for the vending machines themselves (unless I am fundamentally misunderstanding something, and it would not be the first time).

Reply:
Hot coffee is also included here, so it would make sense to add Moffee as well.

https://github.com/space-wizards/space-station-14/blob/838ea2fe2bf0f2650e05610881132f622432bf94/Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/drinks_soda.yml#L37

Reply:
Yeah I just read it again but decided not to comment, my excuse is that I was tired :)

*grumble grumble* non soda-drinks shouldn't be in the soda spawner but it's okay for this PR *grumble grumble*

Reply:
I win this time...